squad/frontend/templates/squad/test_run.html

```diff
@@ -63,64 +63,84 @@
 
 
 {% if tests_status|length > 0 %}
-<h2>Tests</h2>
-<table class='table table-hover table-condensed'>
+    <h2>Tests</h2>
     {% for status in tests_status %}
-    <tr class='warning'>
-        <td>
-            <h4>{{status.suite.slug}}</h4>
-        </td>
-        <td style='vertical-align: middle'>
-            <div class="progress" style='margin-bottom: 0px'>
-                <div class="progress-bar progress-bar-success" style="width: {{status.pass_percentage}}%" title="Pass: {{status.pass_percentage|stringformat:"d"}}%">
-                    <span class="sr-only">Pass: {{status.pass_percentage}}% </span>
+        <a name="tests-{{status.suite}}"></a>
+        <h3>{{status.suite}}</h3>
+        {% with tests=status.tests_by_result %}
+            {% if tests.fail %}
+                {% for test in status.tests_by_result.fail %}
+                    <div class='row fail'>
+                        <div class='col-md-3 col-sm-3'>
+                            <i class='fa fa-times'></i>
+                            {{test.name}}
+                        </div>
+                        <div class='col-md-9 col-sm-9'>
+                            {% if test.log %}
+                                <pre>{{test.log}}</pre>
+                            {% else %}
+                                <em>No log available for this test</em>
+                            {% endif %}
+                        </div>
+                    </div>
+                {% endfor %}
+            {% endif %}
+            {% if tests.skip %}
+                <div class='row'>
+                    <div class='col-md-12 col-sm-12'>
+                        <span class="badge alert-warning" data-placement="top" title="Failures">{{tests.skip|length}} skipped</span>
```

Comment:
title is incorrect

Reply:
fixed (remove title since it doesn't add anything)